src/components/Collections.jsx

```diff
@@ -0,0 +1,110 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { fetchCollections } from "../services/apiService";
+import { Bar } from "react-chartjs-2";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+
+// Registering the necessary components for Chart.js
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+);
+
+function Collections() {
+  const [makersData, setMakersData] = useState({});
+
+  useEffect(() => {
+    const getCollections = async () => {
+      const allCollections = await fetchCollections();
+      const makersCount = {};
+      allCollections.forEach((collection) => {
+        const maker = collection.principalOrFirstMaker;
+        if (maker) {
+          if (!makersCount[maker]) {
+            makersCount[maker] = 0;
+          }
+          makersCount[maker] += 1;
+        }
+      });
+      setMakersData(makersCount);
+    };
+    getCollections();
+  }, []);
+
+  const data = {
+    labels: Object.keys(makersData),
+    datasets: [
+      {
+        label: "Number of Artworks",
```

Comment:
Same with here the label is not at all visible. Use a color that is contrasting with the background color.

Reply:
Addressed, thanks!